nodeup/pkg/model/kube_scheduler.go

```diff
@@ -88,19 +101,66 @@ func (b *KubeSchedulerBuilder) Build(c *fi.ModelBuilderContext) error {
 		})
 	}
 
+	if buildConfigFile {
+		conf, err := b.buildConfigFile()
+		if err != nil {
+			return err
+		}
+		c.AddTask(&nodetasks.File{
+			Path:     KubeSchedulerConfigPath,
+			Contents: fi.NewStringResource(conf),
+			Type:     nodetasks.FileType_File,
+			Mode:     s("0400"),
+		})
+
+	}
+
 	return nil
 }
 
+func (b *KubeSchedulerBuilder) buildConfigFile() (string, error) {
```

Comment:
This sets every setting even if they are not being explicitly configured by Kops, because it's importing the kubeschedulerscheme from upstream.

Would modeling an internal schema closely to what we do for other configurations inside Kops be less heavy handed? It would also allow us to not import yet another upstream dependency.